fabric/src/main/java/io/github/flemmli97/flan/fabric/FlanFabric.java

```diff
@@ -73,6 +74,8 @@ public void onInitialize() {
     public static void serverLoad(MinecraftServer server) {
         Flan.lockRegistry(server);
         ConfigHandler.serverLoad(server);
+        if (FabricLoader.getInstance().isModLoaded("bluemap"))
```

Comment:
Might be better to move this into the mod constructor like dynmap. Is there a reason why its in this event?